policyengine_api/routes/user_profile_routes.py

```diff
@@ -0,0 +1,135 @@
+from flask import Blueprint, Response, request
+from policyengine_api.utils.payload_validators import validate_country
+from policyengine_api.data import database
+import json
+from policyengine_api.services.user_service import UserService
+from werkzeug.exceptions import BadRequest, NotFound
+
+user_profile_bp = Blueprint("user_profile", __name__)
+user_service = UserService()
```

Comment:
__suggestion, non-blocking:__ Rename this to `UserProfileService` (or something similar)

We also have a `user_policy` table that should have its own service in the future, and I wouldn't be surprised if we stored more user-related data in the future. I think it might be advantageous to make the service name more descriptive.